regression/goto-harness/havoc-global-int-01/main.c

```diff
@@ -0,0 +1,8 @@
+int x;
+
+int main()
+{
+  assert(x == 1);
```

Comment:
what exactly is this test testing? Presumably this assertion would still fail if the snapshot harness _didn't_ work?

Reply:
Modified.